List/style.css

```diff
@@ -0,0 +1,9 @@
+.list {
+  list-style-type: None;
+  background-color: white;
+  padding: 30px 20px;
```

Comment:
try using relative units

Reply:
I have updated the units and removed inline styling can you please approve it